### PR TITLE
add documentation for new ldaps properties

### PR DESCRIPTION
**Pull request by passga**
Closes [BR-334](https://bonitasoft.atlassian.net/browse/BR-334)

### CODE REVIEW

File: md/ldap-synchronizer.md [hunk 1]
```diff
@@ -97,8 +97,13 @@ This file defines the LDAP connection settings and specifies the account used fo
 | host\_url | LDAP server URL | ldap://localhost:389 |
 | auth\_type | LDAP authentication type (supported values: none, simple or strong) | simple |
 | principal\_dn | distinguished name (DN) of the user account used for browsing through the LDAP users | cn=Directory Manager |
-| principal\_password | password of the LDAP ?browser account | root |
+| principal\_passwordn  | password of the LDAP ?browser account | root |
```
Comment (educhastenier) [inline md/ldap-synchronizer.md]: `password of the LDAP ?browser account` what does this mean?

File: md/ldap-synchronizer.md [hunk 2]
```diff
@@ -97,8 +97,13 @@ This file defines the LDAP connection settings and specifies the account used fo
...
 | directory\_user\_type | type of the user object ("user" for an Active Directory, "person" for an LDAP) | person |
+| enforce\_ssl | force the connection between ldap client and server to use ssl | false |
+| truststore\_path | allow to configure the java truststore path,  in case of you want to use different keystore than the default  |  |
+| truststore\_password | allow to configure the java truststore password, in case of you want to use a different keystore password than the default  | |
+| truststore\_type | type of the trust store (if different than JKS) | |
```
Comment (educhastenier) [inline md/ldap-synchronizer.md:105]: What values are accepted, other than JKS?

**Review by pabloalonso — COMMENTED** [review]
LGTM

File: md/ldap-synchronizer.md [hunk 3]
```diff
@@ -405,3 +410,21 @@ The LDAP synchronizer will fail if this profile is not defined.
...
+
+
+### LDAPS(TLS) Activation
+**Required** LDAP server should be configured with a valid certificate signed by Certification Authority or with the Auto signed certificate.
+  
+**Configuration:** To configure the LDAP synchronizer for using encrypted connection ( TLS ) :
+
+* Configure the LDAP synchronizer  by editing `ldap.properties` configuration file, as described above
+  * host_url= ldaps://`ldapServerHostname:ldapsServerPort` ( most common `ldapsServerPort` is 636  )
+By default, the `ldapSynchronizer` used the `default java trust Store`, but it is possible to use a custom, by configured the properties :
```
Comment (educhastenier) [inline md/ldap-synchronizer.md]:
```suggestion
By default, the LDAP synchronizer uses the `default java trust Store`, but it is possible to use a custom one, by configuring the properties :
```

File: md/ldap-synchronizer.md [hunk 4]
```diff
@@ -405,3 +410,21 @@ The LDAP synchronizer will fail if this profile is not defined.
...
+  * truststore_path= `locationOfCustomTrustore` 
+  * truststore_password= `passwordOfCustomTrustore` 
+  * truststore_type= `customTrustoreType` ( default JKS)    
+  
+In case of : 
+* When the server Certificate is auto sign (using of custom root certification) you might configured :
+    * the `public certificate` should be imported to the default java or custom trust Store.
```
Comment (educhastenier) [inline md/ldap-synchronizer.md]:
```suggestion
In the following cases:
* when the server certificate is auto-signed (use of custom root certification) you might configure :
    * the `public certificate` should be imported into the default java or custom trust Store.
```

File: md/ldap-synchronizer.md (hunk 3 above)
Comment (passga) [reply md/ldap-synchronizer.md]: ok

File: md/ldap-synchronizer.md (hunk 4 above)
Comment (passga) [reply md/ldap-synchronizer.md]: ok

File: md/ldap-synchronizer.md (hunk 2 above)
Comment (passga) [reply md/ldap-synchronizer.md:105]: it exists a lot of Keystore type, I'm not sure it mandatory to explain all existing type in this documentation. The user could retrieve the information on the internet.

Comment (educhastenier) [reply md/ldap-synchronizer.md:105]: ok, fine to me

**Pull request merged**